stable/home-assistant/templates/deployment.yaml

```diff
@@ -19,6 +19,10 @@ spec:
         app: {{ template "home-assistant.name" . }}
         release: {{ .Release.Name }}
     spec:
+      imagePullSecrets:
```

Comment:
If `pullSecrets` is not defined, there will be an empty `imagePullSecrets ` stanza here.  Is this going to be a problem?  If so, should we have a conditional check to determine if `imagePullSecrets` should be defined at all?

Reply:
In my testing it didn't seem to cause problems, but I agree; it's probably more "correct" to not have the empty stanza. Updated to wrap the `imagePullSecrets` in a `with` block.